src/main/target/STM32F411DISCO/target.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "S411" // STM Discovery F411
+#define USBD_PRODUCT_STRING     "STM32F411DISCO"
```

Comment:
Please change the name of this target to `STM32F411DISCOVERY` - this will make it consistent with the existing targets for STM32 discovery boards.

Reply:
Ideally already existing F4 target would be named `STM32F405DISCOVERY` as well, that's why I wasn't sure what's best, I'll change it.

Reply:
Should the `USBD_PRODUCT_STRING` be the same as target folder name or same as for STM32F4DISCOVERY target (`DISCF4`, so `DISCF411`)?

Reply:
I see, thanks for the explanation!